src/components/organisms/AppRouter/AdminSubrouter.tsx

```diff
@@ -12,73 +12,51 @@ import { AdminAdvancedSettings } from "pages/AdminAdvancedSettings";
 import { AdminDashboard } from "components/organisms/AdminDashboard";
 import { AdminVenueView } from "components/organisms/AdminVenueView";
 
-import { Provided } from "./Provided";
-
 export const AdminSubrouter: React.FC = () => {
   return (
     <Switch>
       {/* Admin OG */}
 
       <Route path="/admin/venue/rooms/:venueId">
-        <Provided withWorldUsers>
-          <RoomsForm />
-        </Provided>
+        <RoomsForm />
       </Route>
 
       <Route path="/admin/venue/creation">
-        <Provided withWorldUsers>
-          <VenueWizard />
-        </Provided>
+        <VenueWizard />
       </Route>
 
       <Route path="/admin/venue/edit/:venueId">
-        <Provided withWorldUsers>
-          <VenueWizard />
-        </Provided>
+        <VenueWizard />
       </Route>
 
       <Route path="/admin/:venueId">
-        <Provided withWorldUsers>
-          <Admin />
-        </Provided>
+        <Admin />
       </Route>
 
       <Route path="/admin">
-        <Provided withWorldUsers>
-          <Admin />
-        </Provided>
+        <Admin />
       </Route>
 
       {/* Admin NG */}
 
       <Route path="/admin-ng/venue/:venueId?/:selectedTab?">
-        <Provided withWorldUsers>
-          <AdminVenueView />
-        </Provided>
+        <AdminVenueView />
       </Route>
 
       <Route path="/admin-ng/advanced-settings/:venueId?/:selectedTab?">
-        <Provided withWorldUsers>
-          <AdminAdvancedSettings />
-        </Provided>
+        <AdminAdvancedSettings />
       </Route>
 
       <Route path={ADMIN_CREATE_SPACE_URL}>
-        <Provided withWorldUsers>
-          <VenueWizardV2 />
-        </Provided>
+        <VenueWizardV2 />
       </Route>
 
       <Route path="/admin-ng/edit/:venueId">
-        <Provided withWorldUsers>
-          <VenueWizardV2 />
-        </Provided>
+        <VenueWizardV2 />
       </Route>
 
       <Route path="/admin-ng">
-        <Provided withWorldUsers>
-          <AdminDashboard />
-        </Provided>
+        <AdminDashboard />
```

Comment:
Should we make all of these inline with Route's `render` or `component` props?

Reply:
However it might look inconsistent because of the other Routes rendering children wrapped with `<Provided>`

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2342/commits/979bc63e9a268bd407d54405dddaab08c0c779b8